_episodes/01-introduction.md

```diff
@@ -16,17 +18,31 @@ keypoints:
 - "CWL supports software containers, supporting reproducibility on different machines"
 ---
 
-# Common Workflow Language
+# Computational Workflows
 
 Computational workflows are widely used for data analysis, enabling rapid innovation and decision making.
-_Workflow thinking_ is a form of "conceptualizing processes as recipes and protocols, structured as workflow or dataflow graphs with computational steps,
-and subsequently developing tools and approaches for formalizing, analyzing and communicating these process descriptions" ([Gryk & Ludascher, 2017](https://doi.org/10.1353/lib.2017.0018)).
+_Workflow thinking_ is a form of "conceptualizing processes as recipes and protocols, structured as workflow or dataflow graphs with computational steps, and subsequently developing tools and approaches for formalizing, analyzing and communicating these process descriptions"[^1].
+Distinct from general business workflows, computational workflows have their primary focus on data and dataflows.
+The workflow managers which are designed for such computational workflows should aid their users with this focus.
+The helpful properties that an ideal computational workflow would possess, to make this possible, are summarised below.
+
+![Summary of handy properties of computational workflows. Categories of properties covered are: composition and abstraction; sharing and adaptability; automation; reporting and accreditation; scalability and infrastructure access; and portability.]({{page.root}}/fig/PropsComputeWF.png){: height="400px"; max-width: 100vw}
+Handy properties for Computational Workflows[^2]
```

Comment:
This sticks out a bit to me. Would it be ok to use the image to link to the slideshare?